### PR TITLE
Build2019 Demo Android ASA Work

### PR DESCRIPTION
**Pull request by andreiborodin**
This PR is to have Android ASA work in Build2019Demo sample:
- Adds ARCore as a submodule to /external folder
- Adds ASA as a submodule to /external folder
- SymLinks ARCore and ASA into the project as appropriate
- Corrects some of the settings to have ARCore and ASA build

It still needs:
- Test and Configure ASA
- Complete instructions for running this demo with ASA

### CODE REVIEW

**Comment by chrisfromwork** [comment]
@julenka I believe the buttons and sliders are already in the solution.

**Comment by julenka** [comment]
> @julenka I believe the buttons and sliders are already in the solution.

Oh yay!!! 🥳 😁 🙌

**Comment by julenka** [comment]
> @julenka I believe the buttons and sliders are already in the solution.

@andreiborodin one thing I noticed is there's no MRTK in this repo so the scripts won't work off the bat. I think the intent is to load them via git submodules? Do you have instructions for how to set that up already in the repo somewhere, or are those coming? I'd like to test out those steps to see if I can figure out how to set up the repo from scratch.

File: samples/Build2019Demo.Unity/README.md [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
+
+## Running the Demo
+
+In order to run the demo, you will need at least two MR/AR capable devices, with the host device preferably a HoloLens 2. Once you have the devices ready, follow the instructions below.
+
+> **ARKit Note:** If you wish to run the experience on an iOS device with ARKit, download the [ARKit repository](https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/downloads/), unzip it, and copy the contents of the `Unity-Technologies-unity-arkit-plugin-94e47eae5954/Assets` folder to `/samples/Build2019Demo.Unity/Assets/`.
+
+### Instructions
+
+These instructions assumes a HoloLens 2 device for host, and an Android phone for spectator. For iOS, replace references to Android with iOS.
```
Comment (chrisfromwork) [inline samples/Build2019Demo.Unity/README.md]: nit: I think host should be replaced with user here to remain consistent with the enum definition.

**Pull request merged**